packages/abi/src/matchers/sway-type-matchers.test.ts

```diff
@@ -26,6 +26,7 @@ const testMappings: Record<keyof typeof swayTypeMatchers, `${string}-matched`> =
   evmAddress: 'evmAddress-matched',
   rawUntypedPtr: 'rawUntypedPtr-matched',
   rawUntypedSlice: 'rawUntypedSlice-matched',
+  str: 'str-matched',
```

Comment:
Not test for this?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3247/commits/ead2f0316ba36a947d9a75bccfa7477b57cc5540